components/generic/SocialShareModal.stories.js

```diff
@@ -0,0 +1,42 @@
+import { storiesOf } from '@storybook/vue';
+import Vuex from 'vuex';
+import SocialShareModal from './SocialShareModal.vue';
+import ShareButton from './ShareButton.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      actions: {
```

Comment:
Add the shareOn key here so that the link titles in the modal are populated correctly.

Reply:
I don't understand, add the shareOn translation key? 
the links in storybook aren't populated correctly on the socialShare component either, for example the facebook link is https://www.facebook.com/sharer/sharer.php?display=page&u=undefined

Reply:
Checking the SocialShare story it uses the store and canonicalUrl, which I assume worked previously, however, now both this and the modal story are not creating the correct link on storybook.

Reply:
ok added the shareOn key